extensions/gitlab/src/components/mr_create.tsx

```diff
@@ -137,6 +140,21 @@ export function MRCreateForm(props: { project?: Project | undefined; branch?: st
     return project?.remove_source_branch_after_merge ?? true;
   };
   const [removeBranch, setRemoveBranch] = useState<boolean | undefined>(undefined);
+  const [selectedTemplateName, setSelectedTemplateName] = useState<string>(NO_TEMPLATE);
+  const [description, setDescription] = useState<string | undefined>(undefined);
+
+  const { data: selectedTemplateDetail } = useCache<TemplateDetail | undefined>(
+    `selected_template_${selectedTemplateName}`,
```

Comment:
The key is accessible from the whole extension so could be that another project has the same template name but a different content. Add something like the `project?.id` to the key to avoid it.

e.g. `selected_template_${selectedTemplateName}_${project?.id}`

Reply:
yeah sure, i don't why but I remove it just before pushing...